tests/lib/internal-rules/internal-no-valid-meta.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * @fileoverview Tests for internal no-valid-meta rule.
+ * @author Vitor Balocco
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/internal-rules/internal-no-invalid-meta"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+
+ruleTester.run("internal-no-invalid-meta", rule, {
+    valid: [
+
+        // context.report() call with no fix
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'some rule',",
+            "            category: 'Internal',",
+            "            recommended: false",
+            "        },",
+            "        schema: []",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {",
+            "            Program: function(node) {",
+            "                context.report({",
+            "                    node: node",
+            "                });",
+            "            }",
+            "        };",
+            "    }",
+            "};"
+        ].join("\n"),
```

Comment:
Let's just stick with ES5 syntax for now. This isn't the place to start adding ES6 syntax that we haven't agreed on or tested out. Using template literals in this way can be part of our larger ES6 discussion.